docs/tutorials/azure/getting-started-azure.md

```diff
@@ -0,0 +1,211 @@
+
+4. Add the credentials to Prowler Cloud
+
+## First Step - Get the Subscription ID:
+
+Got to [Azure Portal](https://portal.azure.com/#home) and search for `Subscriptions` on the bar:
+
+![Search Subscription](./img/search-subscriptions.png)
+
+Get the Azure Subscription ID inside this page:
+
+![Subscriptions Page](./img/get-subscription-id.png)
+
+## Access to Prowler Cloud
+
+1. Go to [Prowler Cloud](https://cloud.prowler.com/) and go inside `Configuration` > `Cloud Providers` page:
+
+    ![Cloud Providers Page](../img/cloud-providers-page.png)
+
+2. Click on `Add Cloud Provider`
+
+    ![Add a Cloud Provider](../img/add-cloud-provider.png)
+
+3. Select `Microsoft Azure`
+
+    ![Select Microsoft Azure](./img/select-azure-prowler-cloud.png)
+
+4. Add the Subscription ID from the first step, input a provider alias (optional) and click on `Next`
+
+    ![Add Subscription ID](./img/add-subscription-id.png)
+
+
+## Setting the Azure Subscription
+
+To make the needed changes on the Subscription configuration you'll need to access to [Azure Portal](https://portal.azure.com/#home)
+
+### Create the Service Principal
+
+To allow Prowler to assume an identity with the necessary privileges to start the scan, a Service Principal must be created. This Service Principal will be used to authenticate with Azure and retrieve the metadata required to perform the scan.
+
+1. Access to Microsoft Entra ID
+
+    ![Search Microsoft Entra ID](./img/search-microsoft-entra-id.png)
+
+2. In the left menu bar, go to "Manage" > "App registrations"
+
+    ![App Registration nav](./img/app-registration-menu.png)
+
+3. Once there, in the menu bar click on "+ New registration" to register a new application
+
+    ![New Registration](./img/new-registration.png)
+
+4. Fill the "Name", select the "Supported account types" and click on "Register". You will be redirected to the applications page.
+
+5. Once in the application page, in the left menu bar, select "Manage" > "Certificates & secrets"
+
+    ![Certificate & Secrets nav](./img/certificates-and-secrets.png)
+
+6. In the "Certificates & secrets" view, click on "+ New client secret"
+
+    ![New Client Secret](./img/new-client-secret.png)
+
+7. Fill the "Description" and "Expires" fields and click on "Add"
+8. Copy the value of the secret, it is going to be used as `AZURE_CLIENT_SECRET` environment variable.
+
+| Value |  |
+| -------- | -------- |
+| Client ID | applicationId for the Service Principal |
+| Client Secret | AZURE_CLIENT_SECRET |
+| Tenant ID | Azure active directory app the tenant Id is shown in the overview |
+
+### Assign the needed Permissions
+
+To allow Prowler to retrieve metadata from the identity assumed and run specific Entra checks, it is needed to assign the following permissions:
+
+    * `Directory.Read.All`
+    * `Policy.Read.All`
+    * `UserAuthenticationMethod.Read.All` (used only for the Entra checks related with multifactor authentication)
```

Comment:
```suggestion
To allow Prowler to retrieve metadata from the identity assumed and run specific Entra checks, it is needed to assign the following permissions:

  * `Directory.Read.All`
  * `Policy.Read.All`
  * `UserAuthenticationMethod.Read.All` (used only for the Entra checks related with multifactor authentication)
```

Less indent to render the list properly while preserving the preformated text.

Reply:
Done